build.gradle

```diff
@@ -14,28 +15,26 @@ gradlePlugin {
   plugins {
     updateChangelog {
       id = "${project.group}.updatechangelog"
-      description = 'A plugin that automatically updates the version in your changelog file.'
       implementationClass = "${project.group}.updatechangelog.UpdateChangelogPlugin"
     }
   }
 }
 
 repositories {
-  jcenter()
+  mavenCentral()
 }
 
 dependencies {
-  compile gradleApi()
-  compile localGroovy()
-  testCompile('org.junit.jupiter:junit-jupiter-api:5.3.1')
-  testRuntime('org.junit.jupiter:junit-jupiter-engine:5.3.1')
+  implementation gradleApi()
+  implementation localGroovy()
 
+  testImplementation('org.junit.jupiter:junit-jupiter-api:5.3.1')
```

Comment:
can't we use the latest version available? https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api/5.7.1

Reply:
Yes we could but the change  would not belong to this PR. I would propose to create a follow-up PR instead with the version bump